tests/unit/pages/categories/Categories.spec.jsx

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import Categories from '../../../../src/pages/categories/Categories'
+import { ModalProvider } from '../../../../src/context/modal-context'
+
+vi.mock('../../../../src/context/modal-context', () => ({
+  useModalContext: () => ({
+    closeModal: vi.fn()
+  }),
+  ModalProvider: vi.fn(({ children }) => <div>{children}</div>) // Mock the provider
+}))
+
+describe('Categories Component', () => {
+  test('renders "Categories" text within the PageWrapper', () => {
+    render(
+      <ModalProvider>
+        <Categories />
+      </ModalProvider>
```

Comment:
We have  `renderWithProviders` function in `tests/test-utils.jsx`, use it for render.